### PR TITLE
[DA-2075] Migration - Normalize informing loop data from Message Broker Event data

### PR DESCRIPTION
**Pull request by dtharpeuno**
## Resolves *[ticket DA-2075]*
https://precisionmedicineinitiative.atlassian.net/browse/DA-2075

## Description of changes/additions
-- The data for the informing _loop records in the MessageBrokerEventData model needs to be normalized and inserted into another future Genomics model for better/faster querying for the new version of the Genomics Outreach API.
-- new informing loop model for Genomics
-- implementation for ingestion via Cloud task from `StoreMessageBrokerEventDataTaskApi` POST

## Tests
- [x] unit tests

### CODE REVIEW

File: rdr_service/genomic/genomic_job_controller.py [hunk 1]
```diff
@@ -387,6 +395,25 @@ def ingest_data_files(self, file_path, bucket_name):
         except RuntimeError:
             logging.warning('Inserting data file failure')
 
+    def ingest_informing_loop_records(self, *, loop_type, records):
+
+        if records:
+            logging.info(f'Inserting informing loop for Participant: {records[0].participantId}')
+
+            module_type = [obj for obj in records if obj.fieldName == 'module_type' and obj.valueString]
+            decision_value = [obj for obj in records if obj.fieldName == 'decision_value' and obj.valueString]
```
Comment (j-kanuch) [inline rdr_service/genomic/genomic_job_controller.py:404]: Is this the portion that normalizes the message broker data? Will the items in `records` always be associated? It looks like `get_informing_loop` returns all records for a `message_record_id` and a `loop_type`. Just checking that the `module_type` and `decision_value` records are linked somehow and that there won't be ambiguity if there are more than the expected items in `records`.

Comment (dtharpeuno) [reply rdr_service/genomic/genomic_job_controller.py:404]: So `records` is a list, that I confirmed will be no longer than 2 via the tech design payload, that is returned in Message Broker Event Dao and will relate back to record in Event Data model via the `message_record_id`.

Comment (j-kanuch) [reply rdr_service/genomic/genomic_job_controller.py:404]: OK, makes sense.

**Review by j-kanuch — APPROVED** [review]

File: rdr_service/api/message_broker_cloud_tasks_api.py [hunk 2]
```diff
@@ -5,53 +5,81 @@
...
+    def call_informing_loop_task(self, obj):
+
+        if GAE_PROJECT != 'localhost' \
+                and 'informing_loop'.lower() \
+                in self.event_type:
+
+            payload = {}
+            informing_records = self.dao.get_informing_loop(
+                obj.messageRecordId,
+                self.event_type
```
Comment (wangy70) [inline rdr_service/api/message_broker_cloud_tasks_api.py:76]: How do you know the records you get is informing records? It seem all events data will be send to you cloud task. Correct me if I miss anything.

Comment (dtharpeuno) [reply rdr_service/api/message_broker_cloud_tasks_api.py:76]: If `informing_loop` is in the event_type parsed from the payload. I can move lines 69-71 to after the loop for conditional test if that makes more sense?

Comment (wangy70) [reply rdr_service/api/message_broker_cloud_tasks_api.py:76]: Never mind! I missed line 70...

**Review by wangy70 — APPROVED** [review]

**Pull request merged**